lib/blockchain_api_web/controllers/hotspot_controller.ex

```diff
@@ -9,8 +9,11 @@ defmodule BlockchainAPIWeb.HotspotController do
   def index(conn, params) do
     hotspots = Query.Hotspot.list(params)
 
-    render(
-      conn,
+    conn
+    |> put_resp_header("surrogate-key", "block")
```

Comment:
There seems to be a lot of duplication with the fastly resp headers? Maybe have some sort of fastly header util? which can put headers based on keys and max-age and use that?

That being said it's nice to be verbose for this, I'm not sure, your call.

Reply:
yeah i was thinking about making some kind of fancy plug, especially for the `?before=x&limit=x` requests. but was just being verbose for now until a pattern emerges